examples/nidcpower_source_dc_voltage/_nidcpower_helpers.py

```diff
@@ -0,0 +1,104 @@
+"""nidcpower Helper classes and functions for MeasurementLink examples."""
+
+from typing import Any, Callable, Dict, Iterable, List, Optional, Tuple, TypeVar
+
+import click
+import grpc
+import nidcpower
+
+import ni_measurementlink_service as nims
+from ni_measurementlink_service import session_management
+from ni_measurementlink_service.measurement.service import MeasurementService
+
+# To use a physical NI SMU instrument, set this to False or specify
+# --no-use-simulation on the command line.
+USE_SIMULATION = True
+
+
+def create_nidcpower_session(
+    measurement_service: MeasurementService,
+    pin_names: Iterable[str],
+) -> Tuple[Any, List[session_management.SessionInformation]]:
+    """Create and reserve nidcpower sessions."""
+    session_management_client = nims.session_management.Client(
+        grpc_channel=measurement_service.get_channel(
+            provided_interface=nims.session_management.GRPC_SERVICE_INTERFACE_NAME,
+            service_class=nims.session_management.GRPC_SERVICE_CLASS,
+        )
+    )
```

Comment:
Rather than duplicating this session management client boilerplate in 7 different _driver_helpers.py files, we should directly address it.

Ideas (mutually exclusive):
- Extract get_session_management_channel()
- Extract create_session_management_client(measurement_service)
- Update the Client class to accept either a grpc.Channel **OR** a MeasurementService

Reply:
Extracted `create_session_management_client(measurement_service)` into helpers.